File: C++/Matrix Validation/Validation of Square and Rectangle matrix only.cpp
#include <iostream>
#include <vector>
using namespace std;

int main(){
    vector <vector<int>> A = {
        { 1, 2, 3 },
        { 4, 5, 6 }
    };
    
    if (A.size() == A[0].size()) {
        cout << "It's a Square matrix" << endl;
    } else {
        cout << "It's a Rectangle matrix" << endl;
    }
}

File: Java/Matrix Validation/Validation of Square and Rectangle matrix only.java
class squareRectangle_Matrix_validation {
    public static void main(String[] args) {
        int[][] A = {
            { 1, 2, 3 },
            { 4, 5, 6 }
        };
        
        if (A.length == A[0].length) {
            System.out.println("It's a Square matrix");
        } else {
            System.out.println("It's a Rectangle matrix");
        }
    }
}